app.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const Sequelize = require('sequelize');
+
+const app = express();
+
+const connection = new Sequelize({
+  database: 'ferretdb',
+  username: 'ferret',
+  password: null,
+  host: 'localhost',
+  dialect: 'postgres',
+  define: {
+    timestamps: false,
+  },
+});
+
+const User = connection.define('user', {
+  firstName: {
+    type: Sequelize.STRING,
+    field: 'first_name',
+  },
+  lastName: {
+    type: Sequelize.STRING,
+    field: 'last_name',
+  },
+  email: {
+    type: Sequelize.STRING,
+    field: 'email',
+    unique: 'true',
+    default: 'salutti',
+  },
+  password: {
+    type: Sequelize.STRING,
+    field: 'password',
+  },
+  rating: {
+    type: Sequelize.DOUBLE,
+    field: 'rating',
+    defaultValue: 0,
+  },
+  active: {
+    type: Sequelize.BOOLEAN,
+    field: 'active',
+    defaultValue: true,
+  },
+});
+
+connection
+  .sync({
+    logging: console.log,
+    force: true, //recreates the entries of the table for testing purposes
+  })
+  .then(() => {
+    console.log("Connection to database worked");
+  })
+  .catch(err => {
+    console.error("Connection to database failed", err);
+  });
+
+const port = process.env.PORT || 3000;
+
+app.post('/api/signup', (req, res) => {
+  User.findOne({ where: { email: req.body.email } })
+  .then(user => {
+    if (user === null) {
```

Comment:
Please create a service that will keep the logic behind creating a user app/services/user.js (maybe later we want to be able to create a user from somewhere else).